jira/resources.py

```diff
@@ -480,34 +543,41 @@ class Issue(Resource):
     """A Jira issue."""
 
     class _IssueFields(object):
+        class _Comment(object):
+            def __init__(self) -> None:
+                self.comments: Optional[List[Comment]] = None
+
         def __init__(self):
-            self.attachment = None
-            """ :type : list[Attachment] """
-            self.description = None
-            """ :type : str """
-            self.project = None
-            """ :type : Project """
-            self.comment = None
-            """ :type : list[Comment] """
-            self.issuelinks = None
-            """ :type : list[IssueLink] """
-            self.worklog = None
-            """ :type : list[Worklog] """
-
-    def __init__(self, options, session, raw=None):
+            self.attachment: Optional[List[Attachment]] = None
+            self.comment = self._Comment()
+            self.description: Optional[str] = None
+            self.issuelinks: Optional[List[IssueLink]] = None
+            self.labels: Optional[List[str]] = None
+            self.project: Optional[Project] = None
+            self.worklog: Optional[List[Worklog]] = None
+
+    def __init__(
+        self,
+        options: Dict[str, str],
+        session: ResilientSession,
+        raw: Dict[str, Any] = None,
+    ):
         Resource.__init__(self, "issue/{0}", options, session)
 
-        self.fields = None
-        """ :type: :class:`~Issue._IssueFields` """
-        self.id = None
-        """ :type: int """
-        self.key = None
-        """ :type: str """
+        self.fields: Optional[Issue._IssueFields] = None
+        self.id: Optional[int] = None
```

Comment:
Referring to both:
https://docs.atlassian.com/software/jira/docs/api/REST/8.13.6/#api/2/issue-getIssue

and 

https://developer.atlassian.com/cloud/jira/platform/rest/v2/api-group-issues/#api-rest-api-2-issue-issueidorkey-get

This Issue id field is a `str`